crates/bevy_render/src/view/visibility/mod.rs

```diff
@@ -81,7 +87,7 @@ impl ComputedVisibility {
     /// [`CoreStage::Update`] stage will yield the value from the previous frame.
     #[inline]
     pub fn is_visible(&self) -> bool {
-        self.is_visible_in_hierarchy && self.is_visible_in_view
+        self.flags.is_all()
```

Comment:
`is_all()` does `Self::all().bits | self.bits == self.bits` which seems like an unnecessary `|` for our usage. We could just do:

```suggestion
         self.flags.bits == ComputedVisibilityFlags::all().bits
```

Reply:
I don't like the readability hit here, but this is probably better for perf.